Jenkinsfile

```diff
@@ -88,7 +91,16 @@ pipeline {
         success {
           script {
             if (params.RELEASE_BUILD && !params.RELEASE_DRY_RUN) {
-              echo env.DOCKER_TAG // For now, just print the deploy tag in the console output -- later, communicate to release metadata service
+              Map buildInfoServer = [
+                "project_id": "${service_server}",
```

Comment:
I'm very skeptical of coercing these values into strings via interpolation. Aren't the associated values already optional strings (i.e., string or null)? I'm very afraid that we'll end up with `"null"` in the database.